pkg/minishift/constants/constants.go

```diff
@@ -30,3 +35,13 @@ const (
 var (
 	ValidIsoAliases = []string{B2dIsoAlias, CentOsIsoAlias, MinikubeIsoAlias}
 )
+
+// ProfileAuthorizedKeysPath returns the path of authorized_keys file in profile dir used for authentication purpose
+func ProfileAuthorizedKeysPath() string {
+	return filepath.Join(constants.Minipath, "certs", "authorized_keys")
+}
+
+// ProfilePrivateKeyPath returns the path of private key of VM present in profile dir which is used for authentication purpose
+func ProfilePrivateKeyPath() string {
```

Comment:
@budhrg  I suggest the function name to be something like `RSAPrivateKeyPath()`. We do not need to keep `profile` name in the function name as it is understood that it needs to be per profile as value of constants.Minipath changes per profile.

Reply:
> We do not need to keep profile name in the function name as it is understood that it needs to be per profile as value of constants.Minipath changes per profile.

It will be more readable from different package and make more sense. `RSAPrivateKeyPath` sounds ambiguous. One need to go to function definition to understand it is actually some file under profile dir.